zsh/completion/_g

```diff
@@ -0,0 +1,2 @@
+#compdef g
+compdef g=git
```

Comment:
This only works post-split, our old setup sourced functions before we loaded `compinit`.